### PR TITLE
app-layer-template: only for version < 7

### PR DESCRIPTION
**Pull request by jasonish**
The C templates have been removed from 7.0.

### CODE REVIEW

File: tests/app-layer-template/test.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ args:
 
 checks:
 - filter:
+    lt-version: 7
```
Comment (catenacyber) [inline tests/app-layer-template/test.yaml:8]: It looks to me that we have twice the same test for `lt-version: 7` and `min-version: 7`
What am I missing ?

**Comment by catenacyber** [comment]
> The C templates have been removed from 7.0.

Have they already ?

File: tests/app-layer-template/test.yaml (hunk 1 above)
Comment (jasonish) [reply tests/app-layer-template/test.yaml:8]: `template.request` and `template.response` differ.

**Comment by jasonish** [comment]
> > The C templates have been removed from 7.0.
> 
> Have they already ?

Premature optimization.  Still requires the PR.

**Comment by catenacyber** [comment]
PR is open https://github.com/OISF/suricata/pull/8182

**Pull request closed without merging**